libs/auth/oauth.go

```diff
@@ -91,11 +91,13 @@ func (a *PersistentAuth) Load(ctx context.Context) (*oauth2.Token, error) {
 	return refreshed, nil
 }
 
-func (a *PersistentAuth) ProfileName() string {
-	// TODO: get profile name from interactive input
+func (a *PersistentAuth) DefaultProfileName() string {
 	if a.AccountID != "" {
 		return fmt.Sprintf("ACCOUNT-%s", a.AccountID)
 	}
+	if a.Host == "" {
+		return "DEFAULT"
+	}
```

Comment:
Does this ever trigger? AFAICT we're always setting a host, right?

Reply:
Yeah, this triggers if you don't specify a `--host` or `[HOST]`. That is, don't specify it on the command line.

Reply:
Before, the default profile name was an empty string.

Reply:
With flag:
```
➜  cli git:(fix/auth-login-host) cli auth login --host https://e2-dogfood.staging.cloud.databricks.com
✔ Databricks profile name: e2-dogfood█
```

Before:
```
➜  cli git:(main) cli auth login
✔ Databricks Profile Name: █
```

After:
```
➜  cli git:(fix/auth-login-host) cli auth login                                                       
✔ Databricks profile name: DEFAULT█
```